meta-oe/recipes-connectivity/linuxptp/linuxptp_3.1.bb

```diff
@@ -17,6 +17,13 @@ export KBUILD_OUTPUT="${RECIPE_SYSROOT}"
 
 do_install() {
     oe_runmake install DESTDIR=${D} prefix=${prefix}
+
+    # Install example configs from source tree
+    install -d ${D}${docdir}/${PN}
+    cp -r ${S}/configs ${D}${docdir}/${PN}
```

Comment:
I wonder if using something like below should be used
```
cp -R --no-dereference --preserve=mode,links
```

Reply:
I have looked in oe-core and meta-openembedded and can not find a common pattern around this. But agree that your suggestion is better if in the future links show up among the installed files.